src/data/mainnet/celo-tokens-info.json

```diff
@@ -653,7 +654,10 @@
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/eXOF.png",
     "isCoreToken": true,
     "name": "ECO CFA",
-    "symbol": "eXOF"
+    "symbol": "eXOF",
+    "isStableCoin": true,
+    "isCashOutEligible": true,
+    "infoUrl": "https://www.dunia.africa"
```

Comment:
I can't find any info on eXOF when i goto this link. Can you provide a more specific url?

Reply:
The website is going through an design update at the moment. I will update this once the new website is ready.